src/coffee/services/authentication.coffee

```diff
@@ -0,0 +1,16 @@
+BaseService = require './base'
+
+# Login service as defined in http://dev.sphere.io/http-api-projects-customers.html#authenticate-customer
+#
+# Example
+#   service = client.authentication
+#   service.save
```

Comment:
Sleeping it over, I think it would make more sense to have this endpoint in the `customers` service
https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/services/customers.coffee

Simply

```coffeescript
login: (body) ->
  unless body
    throw new Error 'Body payload is required for login the customer'
  @_save('/login', body)
```